test/objects.coffee

```diff
@@ -44,6 +44,15 @@ pluck = (x) -> x.a
 eq 100, pluck pluck pluck a: a: a: 100
 
 
+test "Object literal spread", ->
+  obj1 =
+    a: 1
+    b: 2
+  # obj2 = "#{{obj1..., b: 3}}"
+  t1 = CoffeeScript.compile "{obj1..., b: 3}", bare: yes
+  t1 = t1.replace(/\s+/g, "")
+  eq t1, "({...obj1,b:3});"
```

Comment:
Sorry. Long hours :(

Anyway, I couldn't find example how to test the output of `obj2 = {obj1..., b:5}`
CS compiles it to 
```
(function() {
  var obj2;

  obj2 = {
    ...obj1,
    b: 5
  };

}).call(this);
```

but test fails since the syntax {...obj1, b:5} is currently in stage-3 and requires Babel.

To rephrase the question, can Babel be used for tests?